From a11a76a7c22d6731b62ffd174398fc20c3f902dd Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Wed, 17 Jun 2026 09:43:08 +0200
Subject: [PATCH 2/4] v6.18.35-rt5

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index ad3da1bcab7e8..0efe7ba1930e1 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt4
+-rt5
-- 
2.43.0

